### DISCUSSION TITLE
Disable REM Chem when body inactive

### DISCUSSION THREAD

**Discussion by ddyer8** [tagyoureit/nodejs-poolController#494] [Nixie] [answered]
Is there an option to disable REM Chem dosing when the pool body is inactive? I am controlling my AquaRite SWG based on ORP, and it appears it's still commanding dosing when the pool is scheduled off. I don't currently have an additional flow switch available for Nixie - just the one wired to my SWG. I see the wisdom in the extra flow switch and will seek to add, but it seems prudent to only command dosing when a body/pump is running anyway, right?

Thanks for your hard work on this project:)

**Comment by tagyoureit** [comment]
Can you attach a packet capture so we can take a look?

The chlorinator should be wired to a filter relay so it isn't even powered when the pool circuit is off. How is yours wired?

**Comment by ddyer8** [comment]
My SWG and pump are on the same circuit and always hot.  Was counting on the software to command appropriately with AquaRite flow sensor as the safety.

That said, I can put the SWG through a REM relay if that's the preferred mode of control.

I'm not totally sure what should be selected when generating replays.  Please let me know if this has what you need.
[replay.zip](https://github.com/tagyoureit/nodejs-poolController/files/8949422/replay.zip)
.

**Comment by rock-crusher** [comment]
Wire your SWG just as tagyoureit said above!
Funny thing, we just had a discussion about this very issue a few days ago. Below is the link, click on the “view more” option on the bottom left to read all 20 replies. 

https://github.com/tagyoureit/nodejs-poolController/discussions/484#discussioncomment-2958781

**Comment by ddyer8** [comment]
Ok, so I put my SWG on a relay configured in REM.  Then I created a circuit group in Nixie to link the pool body and SWG relay for scheduling.  Is this the recommended configuration?

As a test, REM Chem is commanding my SWG to dose, and I turned my new Pool/SWG group off.  Looks like the software detects that my SWG has gone offline and just freezes the dosing timer.  Is that the expected operation?

**Reply by tagyoureit** [reply]
Pull dashPanel.  We had a restriction on not being able to control an external circuit from the main pool/spa circuits.

You can now just assign the pool circuit to an REM relay, put your SWG on that relay and forego the need to create a circuit group.

![2022-06-22_08-27-26](https://user-images.githubusercontent.com/1646840/175070272-9082fdc8-069c-49af-a62c-fcde01478f9b.png)

> Looks like the software detects that my SWG has gone offline and just freezes the dosing timer. Is that the expected operation?

Yes, when the main pool (or spa) circuit is off then we pause any dosing by the chlorinator (or acid pumps).

**Comment by ddyer8** [comment, marked as answer]
Got it all setup.  Thanks!
